src/Cards.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Cards = (props) => {
+  return (
+    <div class="card" style={{ width: "18rem" }}>
```

Comment:
Remember that you need to convert `class` to `className` when using React

Reply:
done